stock_orderpoint_route/__manifest__.py

```diff
@@ -8,7 +8,7 @@
     "website": "https://github.com/OCA/stock-logistics-warehouse",
     "author": "ForgeFlow, Camptocamp, Odoo Community Association (OCA)",
     "category": "Warehouse",
-    "depends": ["stock"],
+    "depends": ["sale", "stock"],
```

Comment:
Any reason to add this dependency?

Reply:
hello @grindtildeath , thx for your review. It seems to be a mistake, I had tested on a db with pre-installed `sale` module, and if I didn't put `sale` as dependency, the test case would have been failed due to a constraint notnull of `sale_line_warn`. I should have tested on a fresh db instead, I'll update the PR

Reply:
@grindtildeath I pushed the update